packages/wellbeing-mock-data/wellbeing-mock-data.js

```diff
@@ -130,20 +130,89 @@ function createMockActivities() {
   }
 }
 
+
+function getRandomHomeButExcludeCurrent(current_id) {
+  var homeIds = Homes.find().map(function (e) { return e._id; });
+    // remove current home from array
+    var indexOfCurrent = homeIds.indexOf(current_id);
+    if (indexOfCurrent > -1) {
+      homeIds.splice(indexOfCurrent, 1);
+    }
+
+    //return random element from rest of the homes
+    return homeIds[Math.floor(Math.random() * homeIds.length)];
+  }
+  function getRandomMoveInDate(start) {
+    var endDate = new Date();
+    var startDate = new Date();
+    startDate.setMonth(endDate.getMonth() - start);
```

Comment:
Add comments describing, in plain language, what the code is trying to achieve. This makes it much easier to understand your thinking when writing the code.